core/templates/pages/exploration-player-page/learner-experience/input-response-pair.component.html

```diff
@@ -8,8 +8,13 @@
 </ng-template>
 
 <div class="conversation-skin-learner-answer-container">
-  <img class="conversation-skin-user-avatar rounded-circle"
-       [src]="decodedProfilePicture ? decodedProfilePicture : ''" alt="">
+  <picture>
+    <source type="image/webp" [srcset]="profilePictureWebpDataUrl">
+    <source type="image/png" [srcset]="profilePicturePngDataUrl">
+    <img [src]="profilePicturePngDataUrl"
+          class="conversation-skin-user-avatar rounded-circle"
+          alt="">
```

Comment:
Missing alt, please make sure it is everywhere.

Reply:
Done